py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py

```diff
@@ -68,7 +68,8 @@
     "kubeflow/kfctl@HEAD",
     "kubeflow/kubeflow@HEAD",
     "kubeflow/testing@HEAD",
-    "kubeflow/tf-operator@HEAD"
+    "kubeflow/tf-operator@HEAD",
```

Comment:
Is this file still being invoked? I believe we removed this test from pre and postsubmits. Should we go ahead and delete this file?

Reply:
@jlewi Yeah, the file is not invoked for presubmites. Sorry I left behind... how to run the kfctl e2e testing now? use `gcp-blueprints`?

Reply:
The E2E tests were deploying a complete KF instance and then verifying a complete installation was correct.
Those tests were removed because they weren't focused on things directly to kfctl and were causing a lot of pain.

I would suggest we delete this file.

Reply:
OK I will delete the file. Thanks.